go.mod

```diff
@@ -8,29 +8,31 @@ require (
 	github.com/golang/protobuf v1.5.3
 	github.com/google/go-cmp v0.5.9
 	github.com/kubernetes-csi/csi-test/v5 v5.0.0
-	github.com/onsi/ginkgo/v2 v2.9.5
-	github.com/onsi/gomega v1.27.7
-	github.com/prometheus/client_golang v1.15.1
+	github.com/onsi/ginkgo/v2 v2.11.0
+	github.com/onsi/gomega v1.27.10
+	github.com/prometheus/client_golang v1.16.0
 	github.com/prometheus/client_model v0.4.0
-	github.com/prometheus/common v0.42.0
+	github.com/prometheus/common v0.44.0
 	github.com/pseudomuto/protoc-gen-doc v1.5.0
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/viper v1.10.1
 	golang.org/x/sys v0.15.0
 	google.golang.org/grpc v1.58.3
 	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.2.0
 	google.golang.org/protobuf v1.31.0
-	k8s.io/api v0.27.3
-	k8s.io/apimachinery v0.27.3
-	k8s.io/client-go v0.27.3
-	k8s.io/klog/v2 v2.90.1
+	k8s.io/api v0.28.0
```

Comment:
api version is upgraded to v0.28.0. IIUC, This means that the supported Kubernetes version will be changed. 

I thought this should be done in [the Kubernetes upgrade task](https://github.com/topolvm/topolvm/issues/736).

Reply:
I work on the k8s 1.28 support before doing the merge of this PR. After that, I will rebase.
https://github.com/topolvm/topolvm/issues/736

Reply:
I have rebased this PR. Would you review it again?